maven-java/akkaserverless-maven-archetype-event-sourced-entity/pom.xml

```diff
@@ -2,12 +2,12 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>akkaserverless-maven-archetype-event-sourced-entity</artifactId>
-    <version>0.7.0-beta.18</version>
+    <version>0.7.0-beta.18-67-e285d342-SNAPSHOT</version>
```

Comment:
Don't check in specific snapshot version?

Reply:
argh, thanks, I'll remove it again in next PR